packages/react-components/src/components/gif.tsx

```diff
@@ -235,9 +239,8 @@ const Gif = ({
             onMouseLeave={onMouseLeave}
             onClick={onClick}
             onContextMenu={(e: SyntheticEvent<HTMLElement, Event>) => onGifRightClick(gif, e)}
-            ref={container}
```

Comment:
@thighhighcrocs not sure if moving this was intentional, it doesn't look like it would have any effect, but maybe play it safe and leave it on the wrapper element?